src/main/java/com/gr4vy/api/model/PaymentMethod.java

```diff
@@ -345,15 +345,21 @@ public enum MethodEnum {
     GOCARDLESS("gocardless"),
     
     GOOGLEPAY("googlepay"),
+
+    GOOGLEPAY_PAN_ONLY("googlepay_pan_only"),
     
     GOPAY("gopay"),
     
     GRABPAY("grabpay"),
     
     IDEAL("ideal"),
+
+    ID("id"),
```

Comment:
@phillipgr4vy did you manually add this? when I ran the generator earlier this was missing (so the openapi spec needs updating)?

Reply:
@steve-gr4vy the ID shouldnt of been added I've removed it. The googlepay_pan_only I manually added as this came up in the testing in spider. I can add this to openapi spec

Reply:
@phillipgr4vy The model files shouldn't be edited manually, they should be fully managed through the generation script - otherwise manual edits will be required each time the SDK models are updated.  Can we update the openapi spec and re-run the generation script?